src/lib/components/workflow/search-attribute-input/number-input.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+  import NumberInput from '$lib/holocene/input/number-input.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import type { SearchAttributeInputValue } from '$lib/stores/search-attributes';
+
+  export let value: SearchAttributeInputValue;
+  let _value = value ? Number(value) : null;
```

Comment:
Made separate components here for `NumberInput` and `TextInput` to make types happy, but definitely open to ideas on alternatives.